graph/step.go

```diff
@@ -12,6 +12,8 @@ import (
 
 	"github.com/Azure/acr-builder/pkg/image"
 	"github.com/Azure/acr-builder/util"
+	"github.com/docker/distribution/reference"
+	pkgerrors "github.com/pkg/errors"
```

Comment:
remove `pkgerrors` and remove the other `errors` import. Only one and it should cover `.New` and `.Wrap`